### DISCUSSION TITLE
Bug PMP::repair_polygon_soup

### DISCUSSION THREAD

**Discussion by stla** [CGAL/cgal#7009] [Q&A] [answered]
Hello,

Consider this mesh: [cube.off](https://gist.github.com/stla/975fe0981c62b9815d1278081cb4ac11). This is a cube with many duplicated vertices, and 12 distinct triangular faces. If you read it as a polygon soup and apply `PMP::repair_polygon_soup`, then you obtain the 8 vertices as expected, but only 11 faces.

**Comment by sloriot** [comment, marked as answer]
Worked for me, could you share the code used?

**Comment by MaelRL** [comment]
Also, please use https://github.com/CGAL/cgal/issues for bug reports.

**Reply by stla** [reply]
> Also, please use https://github.com/CGAL/cgal/issues for bug reports.

Ok, done. Sorry but I was not 100% sure to have the source of the issue, because I use CGAL via R, and I wanted someone to try.
